backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -0,0 +1,114 @@
+import os
+import json
+
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+""" 
+export APPWRITE_ENDPOINT=<http://localhost/v1>
+export APPWRITE_PROJECT=<project_id>
+export APPWRITE_API_KEY=<api_key>
+"""
+APPWRITE_ENDPOINT = os.environ.get("APPWRITE_ENDPOINT")
+APPWRITE_PROJECT = os.environ.get("APPWRITE_PROJECT")
+APPWRITE_API_KEY = os.environ.get("APPWRITE_API_KEY")
+
+
+client = Client()
+database = Database(client)
+
+(
+    client.set_endpoint(APPWRITE_ENDPOINT)
+    .set_project(APPWRITE_PROJECT)
+    .set_key(APPWRITE_API_KEY)
+)
+
+# # Cleanup all collections created before. Used only in development!
+# listCollection = database.list_collections()
+# for collection in listCollection["collections"]:
+#     result = database.delete_collection(collection_id=collection["$id"])
+
+
+createCollectionResult = database.create_collection(
+    "Announcements",  # Collection Name
+    ["*"],  # Read permissions
+    ["role:team"],  # Write permissions
```

Comment:
This needs to be the Admins team not "role:team".

Reply:
there is no "admin" role I guess: https://appwrite.io/docs/permissions.
Team mean "having privillege": admin, mod, ...
User is normal user

Reply:
We create an "Admins" team using one of the python scripts. You can then specify `team:[TEAM_ID]` to only allow the team with ID TEAM_ID to write to the collection. To get the Team ID you can use the lists endpoint (https://appwrite.io/docs/server/teams?sdk=nodejs-default#teamsList) to get a list with all endpoints and then find the team where the team name is "Admins" to get the ID of the Admins team.

![image](https://user-images.githubusercontent.com/58464184/142294867-85d4082b-1bdd-40d1-8323-798b01d40837.png)

Reply:
See this example for how they give write permission to owners of a team, and read permissions to all members of the team, no matter their role.